single-pool/js/packages/client/src/quarantine.ts

```diff
@@ -0,0 +1,264 @@
+import {
+  getBase58EncodedAddressCodec,
+  Base58EncodedAddress,
+  AccountRole,
+  getProgramDerivedAddress,
+} from '@solana/web3.js';
+
+// HERE BE DRAGONS
+// this is all the stuff that shouldnt be in our library once we can import from elsewhere
+
+function address<TAddress extends string>(string: TAddress): Base58EncodedAddress<TAddress> {
+  return string as Base58EncodedAddress<TAddress>;
+}
```

Comment:
@buffalojoec made one of these for you!

https://github.com/solana-labs/solana-web3.js/blob/9a928cebd68524aabc5e5361da3f3a0f489074cf/packages/addresses/src/base58.ts#L54-L59C2